cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -966,7 +986,7 @@ pub mod bridging {
 					// base delivery fee to local `BridgeHub`
 					Some((
 						XcmBridgeHubRouterFeeAssetId::get(),
-						bp_asset_hub_wococo::BridgeHubWococoBaseFeeInWocs::get(),
+						XcmBridgeHubRouterBaseFee::get(),
```

Comment:
Technically incorrect (`bp_asset_hub_wococo` -> `bp_asset_hub_rococo`), but Wococo has been dropped, so this code will die soon